frontend/proof-of-existence/src/views/certify/CertifyView.vue

```diff
@@ -0,0 +1,145 @@
+<script setup lang="ts">
+import { ref } from "vue";
+import router from "@/router";
+
+const activeTab = ref("first");
+const file = ref(undefined);
+const progress = ref(0);
+const hashing = ref(false);
+const fileName = ref("choose file");
+
+//Methods
+const handleFileUpload = async (e: HTMLInputElement) => {
+  fileName.value = e.target?.files[0].name;
+  file.value = e.target.files[0];
```

Comment:
Type mismatch. e is the event.

```suggestion
const handleFileUpload = async (e: Event) => {
  const target = e.target as HTMLInputElement;
  const targetFiles = target.files;
  const hasTargetFiles = Array.isArray(targetFiles) && targetFiles.length > 0;
  fileName.value = hasTargetFiles ? targetFiles[0].name : defaultFilename;
  file.value = hasTargetFiles ? targetFiles[0] : undefined;
```

Reply:
(if above is accepted, dont forget to set `const defaultFilename = 'choose file'` somewhere above)

Reply:
I did this change, But to this line  " const hasTargetFiles = Array.isArray(targetFiles) && targetFiles.length > 0;", targetFiles are not an array. I think it is an Object